### PR TITLE
Adding Github Action

### PR DESCRIPTION
**Pull request by minhtuev**
Github Actions are the default Github's support for continuous integration; recently we have seen instability with Travis CI.

- Create pythonpackage.yml

### CODE REVIEW

File: .github/workflows/pythonpackage.yml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.5, 3.6]
```
Comment (vijay120) [inline .github/workflows/pythonpackage.yml:18]: 3.7?

File: .github/workflows/pythonpackage.yml [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+      uses: actions/setup-python@v1
+      with:
+        python-version: ${{ matrix.python-version }}
+        java-version: '9.0.4' # The JDK version to make available on the path.
+        java-package: jdk # (jre, jdk, or jdk+fx) - defaults to jdk
+        architecture: x64 # (x64 or x86) - defaults to x64
+    
+    - name: Install dependencies
+      run: |
+        ./scripts/travis_install.sh
```
Comment (vijay120) [inline .github/workflows/pythonpackage.yml:37]: why install travis ci when I assume github actions replicates it?

**Comment by tmehlinger** [comment]
I agree with Vijay's observations. 😄 

I also know _zero_ about Github Actions so if it works, go nuts. At a glance, it looks fine to me.

File: .github/workflows/pythonpackage.yml (hunk 2 above)
Comment (minhtuev) [reply .github/workflows/pythonpackage.yml:37]: This is a poorly named script but basically it installs all of our dependency.

File: .github/workflows/pythonpackage.yml (hunk 1 above)
Comment (minhtuev) [reply .github/workflows/pythonpackage.yml:18]: We had a previous issue with enabling build for Python 3.7 in Travis CI, let me enable it here to see if it works.

**Comment by minhtuev** [comment]
@tmehlinger @vijay120 : Py3.7 is taking too long to complete, let me remove it from the current build for now.

**Pull request merged**